testsuite/features/secondary/min_migrate_to_trad.feature

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# This test case does not test a valid scenario. In SUMA you normally do not
+# delete a traditional client and then register it again as Salt minion. You
+# would always just bootstrap the traditional client as a Salt minion again
+# and Salt would recognize that it is a traditional client and will take the
+# the necessary steps to migrate it
+
+Feature: Migrate a Salt minion into a traditional client
+  As an authorized user
+  I want to migrate these Salt minions to traditional clients and have everything as before
```

Comment:
I think that the core of this feature is more about bootstrapping a salt minion in a unregistered traditional client.
Maybe we can rephrase this line and the title.

Reply:
I adjusted the wording.